internal/rpchelp/helpdescs_en_US.go

```diff
@@ -24,6 +24,14 @@ var helpDescsEnUS = map[string]string{
 	"accountsyncaddressindex-branch":    "Number for the branch (0=external, 1=internal)",
 	"accountsyncaddressindex-index":     "The address index to synchronize to",
 
+	// AccountUnlockedCmd help.
+	"accountunlocked--synopsis": "Report account encryption and locked status",
+	"accountunlocked-account":   "String for the account",
```

Comment:
Use name, not string.  Either "Name of the account" or "Account name" would be fine here.

Reply:
Agreed. Was following the pattern with accountaddressindex-account, accountsyncaddressindex-account, but it did seem weird.